articles/azure-monitor/app/opencensus-python.md

```diff
@@ -142,8 +311,21 @@ First you have to create an Application Insights resource, which will generate a
 
      ![Screenshot of end-to-end transaction interface](./media/opencensus-python/0009-end-to-end-transaction.png)
 
+## View your data with queries
+
+1. You can view the telemetry data that was sent from your application through the Logs(Analytics) blade. 
+
+    ![Screenshot of overview pane with Logs(Analytics) selected in red box](./media/opencensus-python/0010-logs-query.png)
+
+2. For telemetry sent with the Azure Monitor trace exporter, incoming requests will show up under `requests` and out-going/in process requests will show up under `dependencies`.
+
+3. For telemetry sent with the Azure Monitor metrics exporter, metrics sent will show up under `customMetrics`.
+
+4. For telemetry sent with the Azure Monitor logs exporter, logs will show up under `traces` and exceptions will show up under `exceptions`.
+
```

Comment:
The next natural step would be alerting. If we're walking our customers through the data usage scenarios, do we want to end with linking out the page on how to set alerts?

Reply:
I added links similarly to how .NET SDK is doing it. 
https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net